rig/machine_control/common.py

```diff
@@ -0,0 +1,62 @@
+"""Internal-use functions shared between the
+:py:class:`~rig.machine_control.MachineController` and
+:py:class:`~rig.machine_control.BMPController` objects.
+"""
+
+import re
+
+
+VERSION_NUMBER_REGEX = re.compile(r"^(\d+)[.](\d+)[.](\d+)(\D.*)?$")
```

Comment:
Is it not fair to assume that the tags will begin with a "-", that is the spec I think?